nodeup/pkg/model/kubelet_test.go

```diff
@@ -142,3 +152,107 @@ func stringSlicesEqual(exp, other []string) bool {
 
 	return true
 }
+
+func Test_RunKubeletBuilder(t *testing.T) {
+	runKubeletBuilderTest(t, "featuregates")
+}
+
+func runKubeletBuilderTest(t *testing.T, key string) {
+	basedir := path.Join("tests/kubelet/", key)
+
+	clusterYamlPath := path.Join(basedir, "cluster.yaml")
+	clusterYaml, err := ioutil.ReadFile(clusterYamlPath)
+	if err != nil {
+		t.Fatalf("error reading cluster yaml file %q: %v", clusterYamlPath, err)
+	}
+
+	var cluster *kops.Cluster
+	var instanceGroup *kops.InstanceGroup
+
+	// Codecs provides access to encoding and decoding for the scheme
+	codecs := kops.Codecs
+
+	codec := codecs.UniversalDecoder(kops.SchemeGroupVersion)
```

Comment:
Do we really need all this just to call `buildSysConfig`... I think we only need a very few fields to be set here.